Moodis/Moodis/Feature/Login/Register/RegisterForm.cs

```diff
@@ -23,7 +23,7 @@ public partial class RegisterForm : Form
         public RegisterForm()
         {
             InitializeComponent();
-            var loginViewModel = new LoginViewModel();
+            var registerViewModel = new RegisterViewModel();
```

Comment:
Lol, why are we creating a new viewmodel here, when we have already created one in 16 line?

Reply:
Just remove initialization from 16 line, and on this line remove `var`, so this way constructor will create and assign the viewmodel to the property

Reply:
Very good question ⚰️